README.md

```diff
@@ -32,8 +32,7 @@ following next steps:
     - Windows: `yarn windows:migrate`
     - Mac/Linux: `yarn migrate`
 8. Run seeders
-    - Windows: `yarn windows:seed`
-    - Mac/Linux: `yarn seed`
+   `sequelize db:seed:all`
```

Comment:
`yarn windows:seed` and `yarn seed` work now, since they were added in [package.json](https://github.com/MVPWorkshop/substrate-startkit-gui-api/blob/develop/package.json) later

Reply:
👍